libc/benchmarks/gpu/timing/amdgpu/timing.h

```diff
@@ -98,7 +100,9 @@ template <typename F, typename T1, typename T2>
 
   auto result = f(arg1, arg2);
 
-  asm("v_or_b32 %[v_reg], 0, %[v_reg]\n" ::[v_reg] "v"(result) :);
+  asm("v_or_b32 %[v_reg], 0, %[v_reg]\n" ::[v_reg] "v"(
+          static_cast<uint32_t>(result))
+      :);
```

Comment:
Unrelated formatting change? I thin you last colon can be omitted here.

Reply:
Yeah, clang-format says that this line should be split into 2, but you're right about the last colon